<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameSelector</class>
 <widget class="QMainWindow" name="GameSelector">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1185</width>
    <height>658</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Game Selector</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="bgcollectionView">
    <property name="geometry">
     <rect>
      <x>575</x>
      <y>0</y>
      <width>610</width>
      <height>658</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <property name="sortingEnabled">
     <bool>true</bool>
    </property>
    <property name="rowCount">
     <number>1</number>
    </property>
    <property name="columnCount">
     <number>4</number>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>50</number>
    </attribute>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>1</number>
    </attribute>
    <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <row/>
    <column/>
    <column/>
    <column/>
    <column/>
   </widget>
   <widget class="QListWidget" name="mechaniclist">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>511</width>
      <height>391</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdResults">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>140</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>3</number>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Flat</enum>
    </property>
   </widget>
   <widget class="QFrame" name="numPlayerFrame">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>5</y>
      <width>255</width>
      <height>175</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <widget class="QPushButton" name="bestButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>5</y>
       <width>71</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>'Best'</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="numplayerLabel">
     <property name="geometry">
      <rect>
       <x>85</x>
       <y>10</y>
       <width>60</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Players&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="recommendedButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>145</x>
       <y>5</y>
       <width>100</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>'Recommended'</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="Btn2Player">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>55</width>
       <height>55</height>
      </rect>
     </property>
     <property name="text">
      <string>2</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="Btn3Player">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>50</y>
       <width>55</width>
       <height>55</height>
      </rect>
     </property>
     <property name="text">
      <string>3</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="Btn4Player">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>50</y>
       <width>55</width>
       <height>55</height>
      </rect>
     </property>
     <property name="text">
      <string>4</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="Btn5Player">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>50</y>
       <width>55</width>
       <height>55</height>
      </rect>
     </property>
     <property name="text">
      <string>5</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="Btn6Player">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>55</width>
       <height>55</height>
      </rect>
     </property>
     <property name="text">
      <string>6</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="Btn7Player">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>110</y>
       <width>55</width>
       <height>55</height>
      </rect>
     </property>
     <property name="text">
      <string>7</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="Btn8Player">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>110</y>
       <width>55</width>
       <height>55</height>
      </rect>
     </property>
     <property name="text">
      <string>8</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="Btn9Player">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>110</y>
       <width>55</width>
       <height>55</height>
      </rect>
     </property>
     <property name="text">
      <string>9</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="playTimeFrame">
    <property name="geometry">
     <rect>
      <x>265</x>
      <y>5</y>
      <width>255</width>
      <height>175</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <widget class="QLabel" name="PlayTimeLabel">
     <property name="geometry">
      <rect>
       <x>75</x>
       <y>10</y>
       <width>91</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Playing Time</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="Btn30mins">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>55</width>
       <height>55</height>
      </rect>
     </property>
     <property name="text">
      <string>0:30</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="Btn60mins">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>50</y>
       <width>55</width>
       <height>55</height>
      </rect>
     </property>
     <property name="text">
      <string>1:00</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="Btn90mins">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>50</y>
       <width>55</width>
       <height>55</height>
      </rect>
     </property>
     <property name="text">
      <string>1:30</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="Btn120mins">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>50</y>
       <width>55</width>
       <height>55</height>
      </rect>
     </property>
     <property name="text">
      <string>2:00</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="Btn150mins">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>55</width>
       <height>55</height>
      </rect>
     </property>
     <property name="text">
      <string>2:30</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="Btn180mins">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>110</y>
       <width>55</width>
       <height>55</height>
      </rect>
     </property>
     <property name="text">
      <string>3:00</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="Btn240mins">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>110</y>
       <width>55</width>
       <height>55</height>
      </rect>
     </property>
     <property name="text">
      <string>4:00</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="Btn300mins">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>110</y>
       <width>55</width>
       <height>55</height>
      </rect>
     </property>
     <property name="text">
      <string>5:00</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QListWidget" name="categorylist">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>511</width>
      <height>391</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="mechButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>190</y>
      <width>161</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Mechanism</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="catButton">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>190</y>
      <width>161</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Category</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>categorylist</zorder>
   <zorder>numPlayerFrame</zorder>
   <zorder>bgcollectionView</zorder>
   <zorder>mechaniclist</zorder>
   <zorder>lcdResults</zorder>
   <zorder>playTimeFrame</zorder>
   <zorder>mechButton</zorder>
   <zorder>catButton</zorder>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>catButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>categorylist</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>392</x>
     <y>214</y>
    </hint>
    <hint type="destinationlabel">
     <x>652</x>
     <y>240</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mechButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>mechaniclist</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>130</x>
     <y>215</y>
    </hint>
    <hint type="destinationlabel">
     <x>190</x>
     <y>387</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
